blocks/promotion/promotion.js

```diff
@@ -109,6 +109,7 @@ async function buildClock(block) {
   clock.className = 'rolex-frame';
   clock.innerHTML = `<iframe
       id="rolexFrame${placeholders.rolexId}"
+      title="Rolex Promotion"
```

Comment:
This is a note for me to use placeholders here... I forgot we were on Presidents Cup for a minute. 🙃